tests/types/src/valid.ts

```diff
@@ -1574,6 +1603,21 @@ stripe.createPaymentMethod({
   billing_details: {name: '', email: ''},
 });
 
+stripe
+  .collectBankAccountForPayment('', {billing_details: {name: 'Jenny Rosen', email: 'jenny@example.com'}})
+  .then((result: {paymentIntent?: PaymentIntent; error?: StripeError}) => null);
+
+stripe.createPaymentMethod({
+  type: 'us_bank_account',
+  billing_details: {name: '', email: ''},
+});
+
+stripe.createPaymentMethod({
+  type: 'us_bank_account',
+  acss_debit: {institution_number: '', transit_number: '', account_number: ''},
```

Comment:
Should this be here ? Looks like copy/paste from acss.

Reply:
Thanks for the catch! Fixed 👍